README.md

```
@@ -487,6 +527,48 @@ databusclient delete https://databus.dbpedia.org/dbpedia/collections/dbpedia-sna
 docker run --rm -v $(pwd):/data dbpedia/databus-python-client delete https://databus.dbpedia.org/dbpedia/collections/dbpedia-snapshot-2022-12 --databus-key YOUR_API_KEY
 ```
 
+### mkdist command
+
+Create a distribution string from components.
+
+Usage:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)